### PR TITLE
Resolve VCSWP-22701

### PR DESCRIPTION
**Pull request by amahadaya**
Fix test files and SDK file auto-generation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="3.6.1"></a>
+
+## [3.6.1] - 2024-07-31
+
+### Changed
+
+- GetDigit initialTimeoutMS property type change from string to integer
```
Comment (katebrune) [inline CHANGELOG.md:18]: Noticed this is the same as 3.6.0 entry, update to reflect changes

File: apis/DefaultApi.ts [hunk 2]
```diff
@@ -1981,7 +1982,8 @@ export class DefaultApiRequestFactory extends BaseAPIRequestFactory {
 
     /**
      * Make a JWT for WebRTC calling
-      
+     * Make a JWT for WebRTC calling
```
Comment (katebrune) [inline apis/DefaultApi.ts:1985]: duplicate on line 1984

File: __tests__/generatedTests/Dequeue.spec.ts
Comment (katebrune) [inline __tests__/generatedTests/Dequeue.spec.ts:1]: why removed if model still exists?

File: models/TranscribeUtterance.ts [hunk 3]
```diff
@@ -10,9 +10,34 @@
  * Do not edit the class manually.
  */
 
-import { TranscribeUtteranceRecord } from './TranscribeUtteranceRecord';
-import { HttpFile } from '../http/http';
+import { AddToConference } from './AddToConference';
```
Comment (lunderwood-vail) [inline models/TranscribeUtterance.ts:13]: I notice a lot of imports in this file that are unused. Another generator issue?

Comment (lunderwood-vail) [reply models/TranscribeUtterance.ts:13]: Also seeing this in enque.ts, maybe others

**Review by katebrune — APPROVED** [review]

**Pull request merged**